animation.html

```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE html>
```

Comment:
Check whether the design is responsive or not.

Reply:
it's not I haven't used media query